package.json

```diff
@@ -32,15 +32,11 @@
     "ember-data": "^2.12.0-beta.1",
     "ember-export-application-global": "^1.0.5",
     "ember-load-initializers": "^0.6.0",
-    "ember-resolver": "github:201-created/dangerously-set-unified-resolver#isaac/pass-referrer",
-    "ember-source": "github:iezer/ember.js#isaac/referrer-to-resolver",
+    "ember-resolver": "4.2.0",
```

Comment:
@mixonic Thanks for this. This will need https://github.com/ember-cli/ember-resolver/pull/201 to work so can you please update this or make it `~4.2.0` so that it will pick up minor changes to the `ember-resolver`?